UMA-Bots/Configuration-Change/src/hubpool-tests.spec.ts

```diff
@@ -0,0 +1,173 @@
+import { HandleTransaction, TransactionEvent, keccak256 } from "forta-agent";
+import { TestTransactionEvent } from "forta-agent-tools/lib/test";
+import {
+  getFindingInstance,
+  getEventMetadataFromAbi,
+  HUBPOOL_MONITORED_EVENTS,
+  SPOKEPOOL_MONITORED_EVENTS,
+} from "./utils";
+import { provideHandleTransaction } from "./agent";
+import { createAddress, NetworkManager } from "forta-agent-tools";
+import { NetworkDataInterface } from "./network";
+
+const RANDOM_ADDRESSES = [createAddress("0x12"), createAddress("0x54")];
+const TRANSFER_EVENT_ABI = "event Transfer(address,uint)";
+const TEST_HUBPOOL_ADDR: string = createAddress("0x23");
+const TEST_SPOKEPOOL_ADDR: string = createAddress("0x46");
+const MOCK_NM_DATA: Record<number, NetworkDataInterface> = {
+  0: { spokePoolAddr: TEST_SPOKEPOOL_ADDR, hubPoolAddr: TEST_HUBPOOL_ADDR },
+};
+
+describe("Detection of HubPool events on L1", () => {
```

Comment:
```suggestion
describe("Detection of HubPool configuration events on L1", () => {
```

(But I suggest merging the tests)